app/build.gradle.kts

```diff
@@ -10,20 +10,21 @@ plugins {
 }
 
 android {
-    compileSdk = Versions.androidCompileSdkVersion
+    compileSdk = libs.versions.androidCompileSdkVersion.get().toInt()
 
     defaultConfig {
-        applicationId = Packages.name
-        minSdk = Versions.androidMinSdkVersion
-        targetSdk = Versions.androidTargetSdkVersion
-        versionCode = Versions.versionCode
-        versionName = Versions.versionName
+        applicationId = "jp.kztproject.rewardedtodo"
+        minSdk = libs.versions.androidTargetSdkVersion.get().toInt()
+        targetSdk = libs.versions.androidTargetSdkVersion.get().toInt()
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/kseito/RewardedTodo/blob/0a053f19cb2b422d8c2b69cb7cb9e8d853298ad8/app/build.gradle.kts#L18"></span>Not targeting the latest versions of Android; compatibility modes apply. Consider testing and updating this version. Consult the <code>android.os.Build.VERSION_CODES</code> javadoc for details.</td>
    </tr>
  </tbody>
</table>